mappings/net/minecraft/data/server/tag/BlockTagsProvider.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_wayypnpx net/minecraft/data/server/tag/BlockTagsProvider
```

Comment:
Oh, I've touched those providers on #267; I feel like using `BaseBlocksTagProvider` and `FutureBlockTagsProvider` might be better than `BlocksTagProvider` and `UpdateOneTwentyBlockTagsProvider`; i really don't like the mojmap name for the latter (although to be fair, it's only an opinion that might be bad if they add concurrent ones) and you'll might want consistency with the item providers, which had the same split except that they both share a common source that isn't abstract either

Reply:
I'm not entirely sure about the FutureBlockTagsProvider

Even if then in 1.20 there's one for 1.21 stuff, the mapping will be lost due to mojmap having very different names.
I'm also wondering what would happens if there's ever 2 concurrent experiments (which kinda already happened with the combat updates, etc)?

Reply:
yeah, after thinking about it, `FutureBlockTagsProvider` wouldn't really be great; The rest of the feedback might still be worth investigating though, but yeah, maybe `UpdateOneTwenty` is the better name for now (unless Mojang ends revealing that update name :P)